dotcom-rendering/src/components/AdBlockAsk.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { AdBlockAskLeaderboard, AdBlockAskMPU } from './AdBlockAsk';
+
+export default {
+	title: 'Components/AdBlockAsk',
+};
+
+export const Leaderboard = () => {
+	return <AdBlockAskLeaderboard supportButtonHref="#" />;
+};
+
+export const MPU = () => {
+	return <AdBlockAskMPU supportButtonHref="#" />;
```

Comment:
Are we exporting these because they will be used later on in another `.stories` file? Also is the `#` a placeholder that will be replaced with a url?

Reply:
The `export` is required for the components to be picked up by storybook.

And the `#` is just a placeholder for the story, we'll construct a proper URL and pass it in as a prop on the actual site.

Reply:
I didn't know that about storybook. Thank you Chris for the info 👍🏼